src/jit/emitxarch.cpp

```diff
@@ -170,6 +170,14 @@ bool emitter::IsDstDstSrcAVXInstruction(instruction ins)
         case INS_psubusb:
         case INS_psubusw:
         case INS_psubw:
+        case INS_psllw:
```

Comment:
nit: it would help if we would keep alphabetical order of instructions

Reply:
Thanks, will fix.